go/data-source-sql/server.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+// Signavio Workflow connector example that provides a data source for a list of countries.
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"strings"
+	"github.com/signavio/connector-examples/go/data-source-sql/configuration"
+	"github.com/signavio/connector-examples/go/data-source-sql/countries"
+)
+
+var address = ":9000"
+var countriesDatabase *countries.Countries
+
+// Serves a single country.
+func country(response http.ResponseWriter, request *http.Request) {
+	code := strings.TrimPrefix(request.URL.Path, "/country/")
+	country, found := countriesDatabase.FindOne(code)
+	if found {
+		json, _ := json.MarshalIndent(country, "", "  ")
+		response.Header().Set("Content-Type", "application/json")
+		response.Write(json)
+	} else {
```

Comment:
Just stylistic nitpicking here: `response.WriteHeader(http.StatusNotFound)` is pretty :-)

Reply:
Not nitpicking: teaching me the API :)